cmd/infra/powervs/validate.go

```diff
@@ -131,10 +131,40 @@ func validateVpc(vpcName string, resourceGroupID string, v1 *vpcv1.VpcV1) (vpc *
 		return
 	}
 
+	if vpc == nil {
+		err = fmt.Errorf("%s vpc not found", vpcName)
+		return
+	}
+
 	if vpc != nil {
+		vpcId := vpc.DefaultSecurityGroup.ID
+		var e error
```

Comment:
is this variable declaration required here? because we are already using `:=` next line!

Reply:
removed the e and used err itself, since removed the if block.